webpack.common.config.js

```diff
@@ -160,8 +310,12 @@ module.exports = {
     },
 
     resolve: {
+        // Until everything is packaged with webpack, including both the coffee-loader and the babel-loader
+        // is confusing, because the compiled coffeescript files are being picked up by the babel-loader
+        // extensions: ['.js', '.jsx', '.json', '.coffee'],
```

Comment:
This commented line is the same as the uncommented line below it.

Reply:
Yup, meant to remove the `.coffee` part.